### ISSUE TITLE
Masthead:-Accessibility :- User profile popup

### ISSUE THREAD

**Issue by chetankan** [carbon-design-system/ibm-dotcom-library#3156]
Issue Description:- There is no label for the user profile popup
Recommended Fix:- Give label as "user profile.

URL : http://ibmdotcom-web-components-canary.mybluemix.net/?

Happening on :- Voiceover ,iPhone , JAWS FF

**Comment by asudoh** [comment]
I think this has been fixed in the latest build:

![image](https://user-images.githubusercontent.com/1259051/87923286-34fcd700-cab8-11ea-8dda-446b5c61ba4b.png)

Please don't hesitate to speak up if you see otherwise. Thanks!
